### PR TITLE
Add test coverage for Flags enum binding

### PR DESCRIPTION
**Pull request by kichalla**
Fixes: Add test coverage for Flags enum binding in integration tests #3670

@pranavkm @rynowak

### CODE REVIEW

**Comment by kichalla** [comment]
@dougbu

File: test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/SimpleTypeModelBinderTest.cs [hunk 1]
```diff
@@ -189,6 +189,31 @@ public static TheoryData<Type> ConvertableTypeData
...
+            };
+
+            var binder = new SimpleTypeModelBinder();
+
+            // Act
+            var result = await binder.BindModelAsync(bindingContext);
+
+            // Assert
+            Assert.True(result.IsModelSet);
+            Assert.Equal(expected, result.Model.ToString());
```
Comment (rynowak) [inline test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/SimpleTypeModelBinderTest.cs]: Do we need to tostring here? can't we just compare to the enum value? (Pass in expected as an int and cast it)

Comment (kichalla) [inline test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/SimpleTypeModelBinderTest.cs]: :+1:

**Comment by kichalla** [comment]
:up: :date:

**Comment by rynowak** [comment]
:shipit:

**Pull request merged**